concept/answer/ConceptMapImpl.java

```diff
@@ -83,4 +112,97 @@ public boolean equals(Object obj) {
 
     @Override
     public int hashCode() { return map.hashCode();}
+
+    public static class ExplainablesImpl implements Explainables {
+
+        Map<String, Explainable> explainableConcepts;
+
+        Map<Pair<String, String>, Explainable> explainableOwnerships;
+
+        ExplainablesImpl() {
+            this(new HashMap<>(), new HashMap<>());
+        }
+
+        ExplainablesImpl(Map<String, Explainable> explainableConcepts, Map<Pair<String, String>, Explainable> explainableOwnerships) {
+            this.explainableConcepts = explainableConcepts;
+            this.explainableOwnerships = explainableOwnerships;
+        }
+
+        @Override
+        public Explainable concept(String variable) {
+            Explainable explainable = explainableConcepts.get(variable);
+            if (explainable == null) throw new GraknClientException(NONEXISTENT_EXPLAINABLE_CONCEPT, variable);
+            return explainable;
+        }
+
+        @Override
+        public Explainable ownership(String owner, String attribute) {
+            Explainable explainable = explainableOwnerships.get(new Pair<>(owner, attribute));
+            if (explainable == null) throw new GraknClientException(NONEXISTENT_EXPLAINABLE_OWNERSHIP, owner, attribute);
+            return explainable;
+        }
+
+        @Override
+        public Map<String, Explainable> explainableConcepts() {
+            return this.explainableConcepts;
+        }
+
+        @Override
+        public Map<Pair<String, String>, Explainable> explainableOwnerships() {
+            return this.explainableOwnerships;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            ExplainablesImpl that = (ExplainablesImpl) o;
+            return explainableConcepts.equals(that.explainableConcepts) &&
+                    explainableOwnerships.equals(that.explainableOwnerships);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(explainableConcepts, explainableOwnerships);
+        }
+
+
+        static class ExplainableImpl implements Explainable {
```

Comment:
`ExplainablesImpl.ExplainableImpl` feels really nasty - can we rename this class to 
```suggestion
        static class Explainable implements Explainables.Explainable {
```

Reply:
shortened to `ConceptMapImpl.ExplainableImpl`

Reply:
i think this is ok